GTMAppAuth.podspec

```diff
@@ -16,8 +16,8 @@ requests with AppAuth.
   # Note: While tvOS is specified here, only iOS and macOS have support for
   #       obtaining authorization from the user. You can use the classes of
   #       GTMAppAuth with tokens obtained out of band to authorize requests
-  #       on tvOS.
-  s.platforms    = { :ios => "7.0", :osx => "10.11", :tvos => "9.0" }
+  #       on tvOS and watchOS.
+  s.platforms    = { :ios => "7.0", :osx => "10.11", :tvos => "9.0", :watchos => "6.0" }
```

Comment:
Any reason not to go with an earlier min version of watchOS here?  AppAuth currently uses 2.0.

Reply:
IIRC there was some specific API that required watchOS 6 specifically.